pkg/api/apiloader.go

```diff
@@ -102,110 +97,6 @@ func (a *Apiloader) LoadContainerService(
 		curOrchVersion = existingContainerService.Properties.OrchestratorProfile.OrchestratorVersion
 	}
 	switch version {
-	case v20160930.APIVersion:
```

Comment:
after a weekend of wisdon, I think this change in particular (removing from apiloader) is a little premature as once this merge, usage of apiloader in RP needs to change (move the `LoadContainerService()` version specific implementation there too).
this will happen soon, but I need to make sure we are ready on the other side before we go ahead with this PR.

Reply:
This is resolved as we do not use aks-engine's apiloader in the RP anymore.
only the unversioned is used.